### ISSUE TITLE
False-positive wrong-import-order with ipaddress

### ISSUE THREAD

**Issue by sanyassh** [PyCQA/pylint#3924]
### Steps to reproduce
1. file tst_pylint.py:

```
import ipaddress
import re
```

2. pylint tst_pylint.py:

```
... standard import 'import re' should be placed before 'import ipaddress' (wrong-import-order) 
```

### Current behavior
False-positive wrong-import-order error

### Expected behavior
Ordering is correct because both re and ipaddress are in standard library: 
https://docs.python.org/3/library/re.html 
https://docs.python.org/3/library/ipaddress.html

### pylint --version output
pylint 2.6.0
astroid 2.4.2
Python 3.6.1

### P.S.
This also happens with `queue` and `threading` standard modules, giving `standard import "import threading" should be placed before "import queue"`.

**Comment by hippo91** [comment]
@sanyassh thanks for the report. I'm unable to reproduce it. For example linting the following code:
```
#pylint:disable=missing-module-docstring,unused-import
import ipaddress
import re
import astroid
import queue
import threading
```
gives the messages:
```
************* Module bug_pylint_3924
bug_pylint_3924.py:5:0: C0411: standard import "import queue" should be placed before "import astroid" (wrong-import-order)
bug_pylint_3924.py:6:0: C0411: standard import "import threading" should be placed before "import astroid" (wrong-import-order)
```
which is correct. Thus if i lint the code:
```
#pylint:disable=missing-module-docstring,unused-import
import ipaddress
import re
import queue
import threading
import astroid
```
i got no message at all.

```
pylint --version
pylint 2.6.0
astroid 2.4.2
Python 3.6.1 (default, Oct 29 2020, 12:11:58) 
[GCC 8.3.0]
```

**Comment by sanyassh** [comment]
@hippo91 Thank you for the answer. After reading it I went little deeper and found that it was an issue with isort 4.2.5 used internally. After upgrading to isort 5.6.4 there were no false-positives.
